web/crux/.env.example

```diff
@@ -1,5 +1,5 @@
 # Application configs
-KRATOS_ADMIN_URL=http://172.17.0.1:9434
+KRATOS_ADMIN_URL=http://localhost:4434
```

Comment:
But at least we should keep using one of the localhost / 172.x.x.x consistently, it is mixed now.

Reply:
Used localhost every single example file.